src/uu/stty/src/stty.rs

```diff
@@ -182,19 +182,21 @@ fn stty(opts: &Options) -> UResult<()> {
 
     // TODO: Figure out the right error message for when tcgetattr fails
     let mut termios = tcgetattr(opts.file).expect("Could not get terminal attributes");
+    let mut set_arg = nix::sys::termios::SetArg::TCSADRAIN;
 
     if let Some(settings) = &opts.settings {
         for setting in settings {
-            if let ControlFlow::Break(false) = apply_setting(opts.file, &mut termios, setting) {
+            if let ControlFlow::Break(false) =
+                apply_setting(opts.file, &mut termios, &mut set_arg, setting)
```

Comment:
Maybe this could pass `&mut opts` and have `set_arg` live there?

Reply:
I'd say that it's separate from `opts`, because that represents the options as given by user. But it could be bundled with `termios` in a `struct`. Something like:
```rust
struct TtySettings {
    termios: Termios,
    set_arg: SetArg,
    // and any other settings we might need later
}
```

Reply:
Yes new struct sounds best option